test/integration/certificates/generates_new_private_key_per_request_test.go

```diff
@@ -67,16 +69,30 @@ func TestGeneratesNewPrivateKeyIfMarkedInvalidRequest(t *testing.T) {
 				Name: "issuer",
 			},
 			PrivateKey: &cmapi.CertificatePrivateKey{
-				// This doesn't actually make any difference in this test case because there is no existing private
-				// key, meaning there's no private key to re-use.
-				RotationPolicy: cmapi.RotationPolicyAlways,
```

Comment:
> This doesn't actually make any difference in this test case because there is no existing private
> key, meaning there's no private key to re-use.

I've addressed this here and in the test below, by creating the expected Secret containing the expected private key, before creating the Certificate.
So now the test really does test that the private key gets rotated, by default.